v2/models/intervalaction.go

```diff
@@ -0,0 +1,27 @@
+//
+// Copyright (C) 2021 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package models
+
+// IntervalAction and its properties are defined in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/support-scheduler/2.x#/IntervalAction
+// Model fields are same as the DTOs documented by this swagger. Exceptions, if any, are noted below.
+type IntervalAction struct {
+	Timestamps
+	Id           string
+	Name         string
+	IntervalName string
+	Protocol     string
+	Host         string
+	Port         int
+	Path         string
+	Parameters   string
+	HTTPMethod   string
+	User         string
+	Password     string
```

Comment:
Note that in Secure mode the password should not be stored in DB, but instead stored in SecretStore. 
@hutchic,  We need to add this to the Security WG agenda

Reply:
@hutchic , no longer needed as agenda item after @cloudxxx8's clarification.